joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/typestub/TypeStubUtil.scala

```diff
@@ -25,7 +23,7 @@ object TypeStubUtil {
         "."
       }
     }
-    File(Paths.get(fixedDir, "/type_stubs").toAbsolutePath)
+    Paths.get(fixedDir, "/type_stubs").toAbsolutePath.normalize()
```

Comment:
`normalize` only seems necessary because of the extra leading `/` in `/type_stubs`